reference/shipping.v3.yml

```diff
@@ -397,6 +458,24 @@ components:
           readOnly: true
           format: date-time
           example: 2022-09-21T14:15:00+00:00
+    shippingSettings:
```

Comment:
Another item from our equivalent [Tax Settings implementation](https://github.com/bigcommerce/docs/blob/d08d3a38e0eb2eda3eb08a5fd049568f3f2d871a/reference/tax_settings.v3.yml#L54) is that we re-use the same "Tax_Settings" schema object for both request and response, given there is no difference and we expect no difference.

Reply:
Thanks Brett, I've removed `shippingSettingsRequest` and now only `shippingSettings` is used across this file. Looking all correct and clean😅🍻

Reply:
Really nice, cheers!